crates/precompile/src/bls12_381.rs

```diff
@@ -1,18 +1,27 @@
 use crate::PrecompileWithAddress;
 
 mod g1;
+#[cfg(feature = "blst")]
```

Comment:
This does not look great. Can you create one `bls12_381_const.rs` file, scan all bls12 files for constant and move them to that file?